tests/data/TestFont-CFF-compreffor.ttx

```diff
@@ -392,7 +392,7 @@
     <ScriptList>
       <!-- ScriptCount=1 -->
       <ScriptRecord index="0">
-        <ScriptTag value="DFLT"/>
+        <ScriptTag value="latn"/>
```

Comment:
why did "DFLT" become "latn" here? are we no longer using DFLT script for kern feature, not even when we only have a basic non-split lookup?

Reply:
If we find just a single explicit script, we use it.

Reply:
(DFLT is uses only when kerning common against common, because that's how the splitter works)

Reply:
I see. Before, we would use DFLT (implicitly, if declared in global languagesystems) if we had _no_ LTR vs RTL lookup split, or explicitly if we did have two split LTR and RTL lookups but only one set of scripts (either LTR or RTL) was declared in global languagesystems. With this PR, we only use DFLT for the kern_Common lookup containing pairs with undefined script.

The consequence of the change, I suppose, is that kerning would stop working if a text's script isn't set up correctly (e.g. a run of latin mistakenly marked as greek or whatever).
I don't know how desirable it would be to continue supporting the use of DFLT script, at least in the simple scenario when there is only one script-specific lookup (e.g. kern_Latn in addition perhaps to kern_Common). We could in the latter case make a single unsplit `kern` lookup containing all the pairs and don't declare any script/language explicitly so it would be registered under all globally languagesystems, including DFLT (the latter is recommended by the FEA spec and almost always declared).

Reply:
> registering all lookups under DFLT?

hm that might be overkill.. though in theory if we can guarantee the pairs from the split lookups don't overlap each other, it could work. Try, and see if shaping a run of latin text gives you kerning even if text is marked as another script (one that's _not_ in the font's script list)

Reply:
So, I have a lookup like

```fea
feature kern {
    script DFLT;
    language dflt;
    lookup kern_Arab;
    lookup kern_Cyrl;
    lookup kern_Grek;
    lookup kern_Latn;
    lookup kern_Common;
    
    script arab;
    language dflt;
    lookup kern_Common;
    lookup kern_Arab;
    
    script cyrl;
    language dflt;
    lookup kern_Common;
    lookup kern_Cyrl;
    
    script grek;
    language dflt;
    lookup kern_Common;
    lookup kern_Grek;
    
    script latn;
    language dflt;
    lookup kern_Common;
    lookup kern_Latn;
} kern;
```

and change https://github.com/daltonmaag/kerning-validator/blob/main/src/kerning_validator/__main__.py#L212 to always use "Zyyy" or "Abcd"?

Reply:
Actually, it doesn't work on NotoLoopedLaoUI and NotoRashiHebrew when using script=Abcd.